dyn_em/module_advect_em.F

```diff
@@ -3028,7 +3028,6 @@ END SUBROUTINE advect_v
 #endif
 SUBROUTINE advect_scalar   ( field, field_old, tendency,    &
                              ru, rv, rom,                   &
-                             c1, c2,                        &
```

Comment:
Lou,
This is the part I am confused about. Why are the C1, C2 variables removed? They are required for the hybrid vertical coordinate.